File: RetailCore.Shared/Entities/Category.cs
using System;

namespace RetailCore.Shared.Entities;

public class Category: BaseEntity
{
    public string Name { get; set; } = string.Empty;
    public string Slug { get; set; } = string.Empty;
    public string? Description { get; set; }
    public string? ImageUrl { get; set; }
    public Guid? ParentId { get; set; }
    public bool IsActive { get; set; } = true;
    public int SortOrder { get; set; } = 0;
}

File: retail-admin-dashboard/src/types/category.ts
export interface Category {
  id: string; // Guid từ BE
  name: string;
  description?: string;
  sortOrder: number;
  createdDate?: string;
  updatedDate?: string;
}

export interface CreateCategoryRequest {
  name: string;
  description?: string;
  sortOrder: number;
}

export interface UpdateCategoryRequest {
  id: string;
  name: string;
  description?: string;
  sortOrder: number;
}